apps/web/vibes/soul/sections/product-detail/index.tsx

```diff
@@ -283,7 +283,7 @@ function ProductAccordionsSkeleton() {
   );
 }
 
-function ProductDetailSkeleton() {
+export function ProductDetailSkeleton() {
```

Comment:
Export the skeleton so we can use it as a fallback in the [visually editable PDP](https://github.com/bigcommerce/catalyst/pull/1808).